### ISSUE TITLE
Continue deploy also when single file caused error.

### ISSUE THREAD

**Issue by patternpoints** [locomotivecms/wagon#412]
My project has lots of theme asset files, in my current case revloution slider. When i deploy my theme, wagon runs into the error:

```
  persisting revolution/fonts/font-awesome/fonts/fontawesome-webfont.svg [failed]
    Resource invalid: local_path is already taken
```

SInce that file exists already serverside, i dont really care. But the deploy process stopps due to that error. 
It would be very useful if wagon continues to deploy other files and just shows the errormessages related to a specific file.

**Comment by greyskin** [comment]
Yes this is a nice suggestion. It would be nice to be able to choose whether to terminate or continue if there is an error.
